consensus/config.go

```diff
@@ -4,14 +4,15 @@ import "time"
 
 // timeout constant
 const (
-	// The duration of viewChangeTimeout; when a view change is initialized with v+1
-	// timeout will be equal to viewChangeDuration; if view change failed and start v+2
-	// timeout will be 2*viewChangeDuration; timeout of view change v+n is n*viewChangeDuration
-	viewChangeDuration time.Duration = 60 * time.Second
+	// default timeout configuration is shorten to 30 seconds as the consensus is 5s
+	viewChangeTimeout = 30
+	// The duration of viewChangeTimeout for each view change
+	viewChangeDuration time.Duration = viewChangeTimeout * time.Second
 
 	// timeout duration for announce/prepare/commit
-	phaseDuration     time.Duration = 60 * time.Second
-	bootstrapDuration time.Duration = 600 * time.Second
+	// shorten the duration from 60 to 30 seconds as the consensus is 5s
+	phaseDuration     time.Duration = 30 * time.Second
```

Comment:
following on the last PR's comment. How about 45 seconds? We don't want to jump into view change prematurely if retry can fix the transient network problems (we expect most of the problems are transient network issues eventually when we made our protocol code stable)

Reply:
I can adjust it to 45 seconds.  One other thing is the retry frequency, right now, we do retry every 10 seconds if consensus is not reached.  I think we shall shorten the time of retry as well since the consensus is shortened to 5 seconds.   I'll change this to 45 seconds, and retry interval to 7 seconds.